.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,494 @@
+# This code-samples file is used by the MeiliSearch documentation
+# Every example written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
+---
+get_one_index_1: |-
+  let movie_index: Index = client.get_index("movies").await.unwrap();
+list_all_indexes_1: |-
+  let indexes: Vec<Index> = client.list_all_indexes().await.unwrap();
+create_an_index_1: |-
+  let movie_index: Index = client.create_index("movies", Some("movie_id")).await.unwrap();
+update_an_index_1: |-
+  let movie_review_index: Index = client.get_index("movie_review").await.unwrap();
+  movie_review_index.update("movie_review_id").await.unwrap();
+delete_an_index_1: |-
+  movie_index.delete().await.unwrap();
+get_one_document_1: |-
+  let movie: Movie = movie_index.get_document(String::from("25684")).await.unwrap();
+get_documents_1: |-
+  let movie: Vec<Movie> = movie_index.get_documents(None, Some(2), None).await.unwrap();
+add_or_replace_documents_1: |-
+  // Note that the data differs a bit from other examples because we are using the exact same format as the database you downloaded, and the other examples are not
```

Comment:
No need to add this precision from my POV. What do you think @bidoubiwa?

Reply:
It is not needed. Only the getting started is suppose to follow the movie dataset.
For every other example, it is important that it is clear in what it does. Clear enough that everything is understood just by reading the code.